internal/ingress/controller/template/configmap_test.go

```diff
@@ -280,3 +284,32 @@ func TestGlobalExternalAuthSnippetParsing(t *testing.T) {
 		}
 	}
 }
+
+func TestLuaSharedDict(t *testing.T) {
+
+	testsCases := []struct {
+		name   string
+		key    string
+		expect map[string]int
+	}{
+		{
+			name:   "lua valid entry",
+			key:    "lua-shared-dicts",
+			expect: map[string]int{"configuration_data": 5, "certificate_data": 5},
+		},
+
+		{
+			name:   "lua invalid entry",
+			key:    "lua-shared-dic",
+			expect: map[string]int{},
+		},
+	}
+
+	for n, tc := range testsCases {
+		cfg := ReadConfig(map[string]string{tc.key: "configuration_data:5,certificate_data:5"})
```

Comment:
Can you make sure you are testing with different values? To make sure parsing is correct.